<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>InstructionWidget</class>
 <widget class="QWidget" name="InstructionWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="continueButton">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>470</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Continue</string>
   </property>
  </widget>
  <widget class="QPushButton" name="closeButton">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>10</y>
     <width>26</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="controlsButton">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>470</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Controls</string>
   </property>
  </widget>
  <widget class="QPushButton" name="levelInfoButton">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>470</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Level Info</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
